### ISSUE TITLE
Dialog: Alert Dialog content should be read by screen reader when shown

### ISSUE THREAD

**Issue by ambivorous** [primefaces/primeng#11766]
### Describe the feature you would like to see added

The content of the Confirm Dialog and regular Dialog should be read by screen readers.

### Is your feature request related to a problem?

The content of the Confirm Dialog and regular Dialog is not currently read by screen readers.

### Describe the solution you'd like

As described in this example (https://www.w3.org/WAI/ARIA/apg/example-index/dialog-modal/alertdialog.html) another attribute needs to be added to the dialog element: aria-describedby which should be directed to a new ID on the dialog content element in the same way aria-labelledby currently directs to the header element.

**Comment by Adask023** [comment]
I'll try to take care of it.

**Comment by cetincakiroglu** [comment]
Hi,

Accessibility is a major concern of the Prime UI libraries and PrimeNG is no exception. [PrimeTek](https://www.primetek.com.tr/) teams have initiated a significant process to review and enhance the accessibility features of the components. This guide documents the foundation of the general guidelines that PrimeNG will follow and each component documentation will have a separate Accessibility section that states the keyboard support, screen reader compatibility, the implementation details along with tips to achieve WCAG compliancy. This work has been initiated in Q2 2022 and planned to be completed by early Q3. PrimeNG will be the reference implementation which then will be ported to PrimeFaces, PrimeNG and PrimeVue.
